_dashboards/management/scheduled-query-acceleration.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: default
+title: Scheduled Query Acceleration
+parent: Data sources
+nav_order: 18
+has_children: false
+---
+
+# Scheduled Query Acceleration
+Introduced 2.17
+{: .label .label-purple }
+
+Scheduled Query Acceleration (SQA) is designed to optimize direct queries from OpenSearch to Amazon Simple Storage Service (Amazon S3). It addresses issues often faced when managing and refreshing indexes, views, and data in an automated way. 
+
+Query acceleration is facilitated by secondary indexes like [skipping indexes]({{site.url}}{{site.baseurl}}/dashboards/management/accelerate-external-data/#skipping-indexes), [covering indexes]({{site.url}}{{site.baseurl}}/dashboards/management/accelerate-external-data/#covering-indexes), or [materialized views]({{site.url}}{{site.baseurl}}/dashboards/management/accelerate-external-data/#materialized-views). When queries run, they use these indexes instead of directly querying S3. 
+
+The secondary indexes need to be refreshed periodically to stay current with the Amazon S3 data. This refresh can be scheduled using an internal scheduler (within Spark) or an external scheduler.
+
+Using SQA provides the following benefits:
+
+- **Cost reduction through optimized resource usage**: SQA reduces the operational load on driver nodes, lowering the costs associated with maintaining auto-refresh for indexes and views.
+
+- **Improved observability of refresh operations**: SQA provides visibility into index states and refresh timings, offering insights into data processing and the current system state.
+
+- **Better control over refresh scheduling**: SQA allows flexible scheduling of refresh intervals, helping manage resource usage and refresh frequency according to specific requirements.
+
+- **Simplified index management**: SQA enables updates to index settings, such as refresh intervals, without requiring multiple queries, simplifying workflows.
+
+## Concepts
+
+Before configuring SQA, familiarize yourself with the following topics:
+
+- [Optimizing query performance using OpenSearch indexing]({{site.url}}{{site.baseurl}}/dashboards/management/accelerate-external-data/)
+- [Flint index refresh](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md#flint-index-refresh)
+- [Index State Management]({{site.url}}{{site.baseurl}}/im-plugin/ism/index/)
+
+## Prerequisites
+
+Before configuring SQA, verify that the following requirements are met:
+
+- Ensure you're running OpenSearch version 2.17 or later.
+- Ensure you have the SQL plugin installed. The SQL plugin is part of most OpenSearch distributions. For more information, see [Installing plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/).
+- Ensure you have configured an Amazon S3 and Amazon EMR Serverless (needed for access to Apache Spark). 
+
+## Configuring SQA
+
+To configure SQA, perform the following steps.
+
+### Step 1: Configure the OpenSearch cluster settings
```

Comment:
FYI this is enabled by default

Reply:
I put "Set `plugins.query.executionengine.async_query.enabled` to `true` (default value)" so users know it's default. But for `plugins.query.executionengine.async_query.external_scheduler.interval` there's no default value - correct? So users must set it manually.

Reply:
> But for plugins.query.executionengine.async_query.external_scheduler.interval there's no default value - correct? So users must set it manually.

Correct, if this value is empty, from spark side (opensearch-spark) it would have an default value (5 mins)